docs/manual/administrator-handbook.md

```diff
@@ -37,33 +42,66 @@ provides an update online, you might not want to deploy it until
 you've tested it.  This helps ensure that when you upgrade, you are
 getting exactly what you asked for.
 
-## Package layering
+### Hybrid image/packaging via package layering
 
-It is possible to add more packages onto the system that are not part of
-the commit composed on the server. These additional "layered" packages
+It is possible to dynamically add more packages onto the system that are not
+part of the commit composed on the server. These additional "layered" packages
 are persistent across upgrades, rebases, and deploys (contrast with the
-ostree [unlocking](https://github.com/ostreedev/ostree/blob/master/man/ostree-admin-unlock.xml)
-mechanism). This allows you to easily enhance the base set of packages
-on only some machines, or only temporarily (rather than asking to have
-it part of the server compose and affecting every machine). For example,
-you may wish to permanently install some diagnostics tools on a test
-machine.
+ostree [unlocking](https://github.com/ostreedev/ostree/blob/master/man/ostree-admin-unlock.xml) mechanism).
+
+This is where the true hybrid image/package nature of rpm-ostree comes into
+play; you get a combination of the benefits of images and packages.  The
+package updates are still fully transactional and offline.
+
+For example, you can use package layering to install 3rd party
+kernel modules, or userspace driver daemons such as `pcsc-lite-ccid`.
+While most software should go into a container, you have full flexibilty
+to use packages where it suits.
 
 ```
-# rpm-ostree pkg-add <pkg>
+# rpm-ostree install <pkg>
 ```
 
-Will download the target package, its dependencies, and create a new
-deployment with those packages installed.
+Will download the target package, its dependencies, and create a new deployment
+with those packages installed.  To remove layered packages, use `rpm-ostree
+uninstall`.
+
+By default, every `rpm-ostree` operation is "offline" - it has no effect
+on your running system, and will only take effect when you reboot.  This "pending" state is
+called the "pending deployment".  Operations can be chained; for example,
+if you invoke `rpm-ostree upgrade` after installing a package, your new root
+will upgraded with the package also installed.
+
+### Rebasing
 
 ```
-# rpm-ostree pkg-remove <pkg>
+rpm-ostree rebase -b $branchname
 ```
 
-Will create a new deployment with the target package removed.
+Your operating system vendor may provide multiple base branches.  For example,
+Fedora Atomic Host has branches of the form:
+
+ - `fedora/26/x86_64/atomic-host`
+ - `fedora/26/x86_64/updates-testing/atomic-host`
+ - `fedora/27/x86_64/atomic-host`
+
+You can use the `rebase` command to switch between these; this can represent a
+major version upgrade, or logically switching between different "testing"
+streams within the same release. Like every other `rpm-ostree` operation, All
+layered packages and local state will be carried across.
+
```

Comment:
Do you want to mention rebasing to a specific commit here?  Or is that more of an 'advanced' topic?